website/docs/cloud-docs/workspaces/settings/policies.mdx

```diff
@@ -0,0 +1,15 @@
+---
+page_title: Policies - Workspaces - Terraform Cloud
```

Comment:
@tstormk Does this need to be it's own page? Will we be adding more content later? 

If not, a whole page feels unnecessary, and we could instead add a new section in the main workspace settings page (https://github.com/hashicorp/terraform-docs-common/blob/main/website/docs/cloud-docs/workspaces/settings/index.mdx) to cover this and be good to go as @hcrhall suggested. What do you think?